lms/djangoapps/class_dashboard/tests/test_dashboard_data.py

```diff
@@ -85,6 +98,36 @@ def setUp(self):
                     state=json.dumps({'attempts': self.attempts}),
                 )
 
+            # adding an instructor and a staff to the site. They should not be counted. Test should not fail
+            StudentModuleFactory.create(
+                grade=1,
+                max_grade=1,
+                student=instructor_member,
+                course_id=self.course.id,
+                module_state_key=self.item.location,
+                state=json.dumps({'attempts': self.attempts}),
+            )
+
+            StudentModuleFactory.create(
+                grade=1,
+                max_grade=1,
+                student=staff_member,
+                course_id=self.course.id,
+                module_state_key=self.item.location,
+                state=json.dumps({'attempts': self.attempts}),
+            )
+
+            # Adding this student will fail the test
+
+            # StudentModuleFactory.create(
+            #     grade=1,
+            #     max_grade=1,
+            #     student=newstudent,
+            #     course_id=self.course.id,
+            #     module_state_key=self.item.location,
+            #     state=json.dumps({'attempts': self.attempts}),
+            # )
+
```

Comment:
The commented-out test was actually just notes for myself. I was testing how to add a user via StudentModuleFactory.create().  I guess I should just remove it.  It was not meant to be part of the automated test.